lib/config/secrets.ts

```diff
@@ -85,21 +85,28 @@ function replaceSecretsInString(
 
 function replaceSecretsinObject(
```

Comment:
@zharinov We should use  uppercase for `i` in a follow up PR. 😉
```suggestion
 function replaceSecretsInObject(
```

Reply:
https://github.com/renovatebot/renovate/issues/13861